src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -89,7 +94,12 @@ type WorkspaceItem = ListItem &
     };
 
 // eslint-disable-next-line react/no-unused-prop-types
-type GetMenuItem = {item: WorkspaceItem; index: number};
+type GetWorkspaceMenuItem = {item: WorkspaceItem; index: number};
+
+type DomainItem = ListItem & {title: string; action: () => void; disabled: boolean};
+
+// eslint-disable-next-line react/no-unused-prop-types
+type GetDomainMenuItem = {item: DomainItem; index: number};
```

Comment:
Is this `eslint` rule necessary? I see that we use both `item` and `index` later in the codebase. Besides, do we need to pass `index` to `GetWorkspaceMenuItem` if we disable the rule that checks if all props get used?